rclpy/rclpy/__init__.py

```diff
@@ -56,16 +58,20 @@ def create_node(node_name):
     return Node(node_handle)
 
 
-def spin_once(node):
+def spin_once(node, timeout_sec=None):
     wait_set = _rclpy.rclpy_get_zero_initialized_wait_set()
 
     _rclpy.rclpy_wait_set_init(wait_set, len(node.subscriptions), 0, 0)
 
     _rclpy.rclpy_wait_set_clear_subscriptions(wait_set)
     for subscription in node.subscriptions:
         _rclpy.rclpy_wait_set_add_subscription(wait_set, subscription.subscription_handle)
+    if timeout_sec is None:
+        timeout = -1
+    else:
+        timeout = int(timeout_sec * S_TO_NS)
```

Comment:
`int()` only needs to be called on the parameter `timeout_sec` (not on the know factor / constant).